projects/angular-grid-layout/src/lib/utils/grid.utils.ts

```diff
@@ -147,8 +145,9 @@ export function ktdGridItemResizing(gridItemId: string, config: KtdGridCfg, comp
         h: screenYPosToGridValue(height, config.rowHeight, gridElemClientRect.height)
     };
 
-    layoutItem.w = Math.max(1, layoutItem.w);
-    layoutItem.h = Math.max(1, layoutItem.h);
+    layoutItem.w = limitNumberWithinRange(layoutItem.w, layoutItem.minW, layoutItem.maxW);
+    layoutItem.h = limitNumberWithinRange(layoutItem.h, layoutItem.minH, layoutItem.maxH);
```

Comment:
We should avoid setting a 'w' or 'h' less than 1. If the minW or minH are less than 1 we should use 1 as a min value.